Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -207,6 +207,16 @@ Ref<WebProcessProxy> WebProcessProxy::createForRemoteWorkers(RemoteWorkerType wo
     return proxy;
 }
 
+#if ENABLE(WEBCONTENT_CRASH_TESTING)
+Ref<WebProcessProxy> WebProcessProxy::createForWebContentCrashy(WebProcessPool& processPool)
+{
+    auto proxy = adoptRef(*new WebProcessProxy(processPool, nullptr, IsPrewarmed::No, CrossOriginMode::Shared, LockdownMode::Disabled));
+    proxy->setCrashyProcess();
```

Comment:
I think we usually would call this `setIsCrashyProcess()`, with the getter being `isCrashyProcess()`

Reply:
I'll make this change.